include/seastar/core/smp.hh

```diff
@@ -324,6 +324,7 @@ class smp : public std::enable_shared_from_this<smp> {
     static thread_local smp_message_queue**_qs;
     static thread_local std::thread::id _tmain;
     bool _using_dpdk = false;
+    std::vector<unsigned> _shard_to_numa_node_mapping;
```

Comment:
It would make more sense to return this shard's mapping rather than the entire smp. You can add an accessor to return smp instead.

Reply:
This comment belongs in reactor.hh, not here

Reply:
Done in v3.